.changeset/fantom-deprecation.md

```diff
@@ -0,0 +1 @@
+Deprecate fantom
```

Comment:
Add this so it works correctly:

```
---
'@tenderly/sdk': minor
---

Deprecate fantom
```

Example from [this PR](https://github.com/Tenderly/tenderly-sdk/pull/127/files):
<img width="1158" alt="image" src="https://github.com/user-attachments/assets/566b4483-fa53-4dd1-a6dc-462b33c1646d" />